cypress/integration/login.js

```diff
@@ -1,28 +1,35 @@
+import LoginPage from '../object/LoginPage'
+import TimeLoggingPage from '../object/TimeLoggingPage'
 const testUser = 'TestCon User 1'
 
 describe('Login functionality', function() {
     it('Should display validation for empty user after attempted loggin', function () {
-       
-        cy.visit('/')
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click()
-        cy.get('.Select.not-valid').should('be.visible')
+        const loginPage = new LoginPage()
```

Comment:
You could extract this variable outside of the describe block and put it next to `testUser` so that you don't have to recreate it again later